examples/configuration.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: pkg.crossplane.io/v1
 kind: Configuration
 metadata:
-  name: platform-ref-gcp
+  name: upbound-platform-ref-gcp
 spec:
-  package: xpkg.upbound.io/upbound/platform-ref-gcp:v0.1.0
+  package: xpkg.upbound.io/upbound/platform-ref-gcp:v0.2.1
   packagePullSecrets:
     - name: package-pull-secret
```

Comment:
Do we need package pull secrets still?

Reply:
good point but we will still need them after adding the documentation chunk similar to https://github.com/upbound/platform-ref-aws/#using-your-custom-platform